fai/fai_test.go

```diff
@@ -85,6 +85,33 @@ TTCATTAATCATTTGTACTGGATCTGTTCG
 		},
 		{
 			in: `>FN654386.1
+TTTTTCAAAGACGTTAAGAGCATCAAACAGAATCATTTTGTTCTCGGATGAGAAGCTGAA
+TCGTGTTGCTTCTCGGTCATACCAAAGACC
+>
+GCGAACTGATGGTCAAGCACAGCTAGAACGTCTTCGTTTGAAGCTGGAGACGATTGCCGCGCAGGCAACT
+CGCCGACATCGACGATGTCTTCGTAGTCAT
+>FN654386.3
+CTGAATTGAGCCGCGGGCGATTGATTCGTGTCGGCGCGTCAGGAGGAAGTTCAAGTCGGAATCTCGCGTT
+TTCATTAATCATTTGTACTGGATCTGTTCG
+`,
+			idx: nil,
+			err: errors.New("fai: missing sequence name at 104"),
+		},
+		{
```

Comment:
Why are there two identical tests? (`idx` is nil in the second implicitly).

Reply:
The second test has only spaces for the name. I'm happy to remove that test though.

Reply:
Resolved in 25658fd85b0583d425debc45806b89a25d9c61b0

Reply:
That's pretty brittle and not very clear. If that's the intention, please put back the explicit `idx: nil` so that its not confounding, and include some syntax that shows that there is an explicit set of spaces. Something like
```
>`+"    "+`
```

Reply:
Will do

Reply:
Re-added in 6babe23